core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultNode.java

```diff
@@ -180,7 +180,7 @@ public NodeMetricUpdater getMetricUpdater() {
 
   @Override
   public String toString() {
-    return String.format("%s(%s,%s)", super.toString(), hostId, endPoint);
+    return String.join("@", getHostId().toString().trim(), getEndPoint().toString().trim());
```

Comment:
I would've preferred to have this as a default method on the interface... but that's not allowed. :(

Reply:
Shouldn't we have a unit test that is testing that behavior?

Reply:
Hmmmm, good question.  My original inclination was to say we probably don't need one since the toString() impl doesn't really affect functionality.  But I don't see how it _hurts_ anything, and it was important enough to file a ticket to address in the first place... so yeah, maybe we should.